src/interfaces/player.ts

```diff
@@ -1,20 +0,0 @@
-export interface Outfit {
-  readonly members: Array<Player>;
-  readonly name: string;
-  readonly id: string;
-  readonly created_at: Date;
-  /**
-   * Must be between 0 and 4 chars
-   */
-  readonly tag: string;
-}
-
-export interface Player {
```

Comment:
For some types of players, the population estimates may provide accurate positioning information of players. This information is not available through the API to avoid hunting of players or squads.

If we introduce leaderboards or other player-specific information down the road, this would be done through a separate endpoint; still not allowing looking up players by name.